tests/test_app.py

```diff
@@ -0,0 +1,99 @@
+# tests/test_app.py
+
+from http.client import responses
+import unittest
+import os
+import sys
+import logging
+
+os.environ['TESTING'] = 'true'
+
+log = logging.getLogger(__name__)
+
+from app import app
+
+class AppTestCase(unittest.TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+
+    def test_home(self):
+        response = self.client.get("/")
+        assert response.status_code == 200
+        html = response.get_data(as_text=True)
+        assert "<title>Home</title>" in html
+
+        assert '<img src="./static/img/lucas/lucas.jpg" class="d-block mx-lg-auto img-fluid rounded-circle" alt="Bootstrap Themes" width="700" height="500" loading="lazy">' in html
```

Comment:
Add more tests here if you can, because you are only testing for 2 items inside your HTML. @mannynacc